modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -76,7 +76,7 @@ $ oc adm upgrade --to-latest
 +
 [source,terminal]
 ----
-Updating to latest version 4.11.18
+Updating to latest version 4.<y+1.z>
```

Comment:
Earlier you had `eus-<4.y+2>`.  I usually use `eus-4.(y+1)` to show that `y+1` is all part of that minor-version slot.  I'm agnostic about how you want to represent this, but we probably want to be consistent about whether or not the major-version `4` is inside or outside the group-marking delimiters.

Reply:
Agreed :) @shellyyang1989 suggested as well. Updated!